js/src/weekly_report.js

```diff
@@ -0,0 +1,60 @@
+// # Generate a weekly report of completed issues per repo
+const fetch = require('node-fetch')
+const {
+  compose,
+  zipObj,
+  tap,
+  mapAwait,
+  tapLog,
+  prop,
+  gte,
+  map,
+  filter,
+  branch,
+  entries,
+} = require('./utils')
+
+// check for arg
+const date = process.argv?.[2]
+if (!date) {
+  throw new Error(
+    'Please call this script with a date in the YYYY-MM-DD format.'
+  )
+}
+
+// data
+const repos = [
+  'openverse',
+  'openverse-frontend',
+  'openverse-api',
+  'openverse-catalog',
+]
+
+// internals
+const API_ROOT = `https://api.github.com`
+const makeIssueUrl = owner => repo =>
+  `${API_ROOT}/repos/${owner}/${repo}/issues?state=closed&since=${date}&per_page=10000`
+const makeWpIssueUrl = makeIssueUrl('wordpress')
+// this key only has public repo access, i don't care about showing it
+const authFetch = url =>
+  fetch(url, {
+    headers: {
+      Authorization:
+        'Basic emFja2tyaWRhOmdocF9JVHdJM2ZGUkVqSjZkOWRBUFFBOE5pWktwNlU0dXkxRTVFak8=',
+    },
+  })
+const fetchRepoIssues = compose(authFetch, makeWpIssueUrl)
+const getJSON = i => i.json()
+const filterClosed = compose(gte(date), prop('closed_at'))
```

Comment:
Currently, this report shows all closed issues. This means that the issue we closed manually, without a solution (like `[#42 Provide HTML code to allow CC Search to be embedded in other websites (original #505)](https://github.com/WordPress/openverse-frontend/issues/42)`) are also in the report.
We could (I think?), instead, only look at the issues that have `pull_request` property.

Reply:
> This means that the issue we closed manually...are also in the report.

I'm actually content with that. Those issues being closed can still provide useful information to contributors.